tsconfig.json

```diff
@@ -5,7 +5,12 @@
     "outDir": "out",
     "lib": ["es6"],
     "sourceMap": true,
-    "rootDir": "src"
+    "rootDir": "src",
+    "baseUrl": "./",
+    "paths": {
+      "*": ["types/*"]
+    },
+    "typeRoots": ["node_modules/@types", "node_modules/", "types"]
```

Comment:
I believe searching for types in all packages under `node_modules/`, is what's causing the error here.